packages/ketcher-core/src/application/editor/modes/SequenceMode.ts

```diff
@@ -883,6 +941,14 @@ export class SequenceMode extends BaseMode {
     return modelChanges;
   }
 
+  private showMergeWarningModal(editor: CoreEditor) {
```

Comment:
I think it's rather an error than a warning message (which I covered by a pop-up in the bottom of the editor but doesn't block it)